tests/wpt/web-platform-tests/cssom-view/offsetPropertiesInline.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>cssom-view - offsetParent, offsetTop, offsetLeft, offsetWidth, and offsetHeight on inline elements</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  #real-offset-parent {
+    font: 10px/1 Ahem;
+    margin: 0;
+    border: none;
+    padding: 0;
+  }
+  #real-offset-parent, #decoy-offset-parent {
+    position: relative;
+  }
+</style>
+<body>
+<div id="real-offset-parent">
+  <span id="inline-1">ABC</span>
+  <span id="inline-2">ABC<br />ABC</span>
+  <span id="inline-3">ABC</span>
+</div>
+<div id="decoy-offset-parent"></div>
+<script>
+  setup({ explicit_done: true });
```

Comment:
instead of `explicit_done`, you can use:

```js
var t = async_test("...");
window.onload = t.step_func_done(function() {
    // ...
});
```

but it's fine this way too if you prefer it.